app/models/openingtime.rb

```diff
@@ -14,6 +14,19 @@ class Openingtime < ApplicationRecord
   validates :closes, presence: true
   validate :opens_before_closes
 
+  def day_as_string
+    mapping = {
+      0 => 'Monday',
+      1 => 'Tuesday',
+      2 => 'Wednesday',
+      3 => 'Thursday',
+      4 => 'Friday',
+      5 => 'Saturday',
+      6 => 'Sunday'
+    }
+    mapping[day]
+  end
+
```

Comment:
just used in tests? then, it should be a spec helper (better: use it!)

Reply:
We already use it in the show-view for locations :)